docs/getting_started_jax.ipynb

```diff
@@ -0,0 +1,669 @@
+        "\n",
+        "Also, this will be relaxed in the JAX side soon, but you will be able to use `UNSPECIFIED` on in/out specs and have `auto` axes propagate from inside to outside the shard_map body (and vice-versa)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "_UfLgKHie0ai"
+      },
+      "source": [
+        "### Auto partitioners\n",
+        "\n",
+        "In progress."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "JTR9tmEJnM6v"
+      },
+      "source": [
+        "### XLA_DUMP_TO\n",
+        "\n",
+        "When specifying the `XLA_DUMP_TO`, you will see an additional `shardy/` directory containing various dumps of the StableHLO program. A lot of them are currently only relevant to the Shardy team to debug issues. The one you should focus on when debugging is `sdy_module_after_sdy_export.mlir` which is the module after propagation finishes and the StableHLO program.\n",
+        "\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0kIyBosUoT_j"
+      },
+      "source": [
+        "## 2. Why Shardy?"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3EdCWagVNiuF"
+      },
+      "source": [
+        "### Readability\n",
+        "\n",
+        "As seen above, it's much easier to read the shardings and shard_maps and understand how they match what is happening in the JAX code. Similarly GSPMD propagation will give back HLO code - not MLIR which both Shardy and `jax.jit.lower` return."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "si1rYISKoYNO"
+      },
+      "source": [
+        "### Interpretability\n",
+        "\n",
+        "We are planning on exposing a feature we call \"user priorities\" (not in JAX yet!). It allows you to attach a value telling Shardy how important a tensor's dimension sharding is over other constraints in the program.\n",
+        "\n",
+        "Higher priorities are defines as lower values (lowest being 0).\n",
+        "\n",
+        "```python\n",
+        "PartitionSpec(None, 'x', 'y', priorities=(None, 0, 1))\n",
+        "```\n",
+        "\n",
+        "Here the sharding of dim 1 on `x` has a higher priority than dim 2 on `y`, meaning dim 1 will be propagated through the program first and then dim 2, meaning any potential sharding conflicts will be explicitly resolved by having `x` propagated first.\n",
+        "\n",
+        "This can be helpful for debugging models as well by having you break down your sharding strategies to separate rounds of propagation in Shardy. For example:\n",
+        "\n",
+        "* Priority 0: data parallelism\n",
+        "* Priority 1: megatron\n",
+        "* Priority 2: ZeRO sharding\n",
+        "\n",
+        "\n",
+        "(Interpretability: easier to read shardings, MLIR, user priorities)"
```

Comment:
Remove these three lines (I think since the line above has a `\n` it should be fine)